Tests/PackageLoadingTests/PD_5_3_LoadingTests.swift

```diff
@@ -505,17 +505,6 @@ final class PackageDescription5_3LoadingTests: PackageDescriptionLoadingTests {
 
     func testNonZeroExitStatusDoesNotAssert() async throws {
         let content = """
-            #if canImport(Glibc)
-            import Glibc
-            #elseif canImport(Musl)
-            import Musl
-            #elseif os(Windows)
-            import MSVCRT
-            import WinSDK
```

Comment:
@hyp, the Windows CI passed but I notice it doesn't run the SPM tests, just builds the source? Let me know if this change is okay on Windows.